### PR TITLE
Fix broken links on UCP links and TOC on 2.1

### LINKED ISSUE #3533: Broken link at bottom of page for "Use externally-signed certificates"
File: [datacenter/ucp/2.1/guides/admin/configure/integrate-with-dtr.md](https://docs.docker.com/datacenter/ucp/2.1/guides/admin/configure/integrate-with-dtr/), CC @joaofnfernandes

### PR DESCRIPTION
**Pull request by londoncalling**
I haven't got this working yet, but stay tuned, almost there.

### What's changed

- Fixed broken links in "what's next" sections
- Added missing topic to TOC on v2.1

### Related

Fixes #3533

### Reviewers

@grahamnscp

Signed-off-by: Victoria Bialas <victoria.bialas@docker.com>

### CODE REVIEW

File: _data/toc.yaml [hunk 1]
```diff
@@ -1572,6 +1572,8 @@ manuals:
               title: Use domain names to access services
             - path: /datacenter/ucp/2.1/guides/admin/configure/run-only-the-images-you-trust/
               title: Run only the images you trust
+            - path: /datacenter/ucp/2.1/guides/admin/configure/integrate-with-dtr/
```
Comment (joaofnfernandes) [inline _data/toc.yaml]: Can you move this to the 'Configure' section (around line 1551). It makes more sense from an information architecture standpoint.

File: datacenter/ucp/2.1/guides/admin/configure/integrate-with-dtr.md [hunk 2]
```diff
@@ -139,4 +139,4 @@ steps as you used to configure your local computer.
 
 ## Where to go next
 
-* [Use externally-signed certificates](index.md)
+* [Use your own TLS certificates](use-your-own-tls-certificates/.md)
```
Comment (joaofnfernandes) [inline datacenter/ucp/2.1/guides/admin/configure/integrate-with-dtr.md]: I think you have an extra slash here.

File: datacenter/ucp/2.2/guides/admin/configure/integrate-with-dtr.md [hunk 3]
```diff
@@ -140,4 +140,4 @@ steps as you used to configure your local computer.
 
 ## Where to go next
 
-* [Use externally-signed certificates](index.md)
+* [Use your own TLS certificates](use-your-own-tls-certificates/.md)
```
Comment (joaofnfernandes) [inline datacenter/ucp/2.2/guides/admin/configure/integrate-with-dtr.md]: Also an extra slash here.

File: datacenter/ucp/2.1/guides/admin/configure/integrate-with-dtr.md (hunk 2 above)
Comment (londoncalling) [reply datacenter/ucp/2.1/guides/admin/configure/integrate-with-dtr.md]: omg, thanks!

**Pull request merged**